python/PyQt6/gui/auto_generated/elevation/qgselevationcontrollerwidget.sip.in

```diff
@@ -59,6 +59,15 @@ Returns a reference to the slider component of the widget.
 %Docstring
 Returns a reference to the widget's configuration menu, which can be used
 to add actions to the menu.
+%End
+
+    double fixedRangeWidth() const;
```

Comment:
To my non-native English brain, fixed range width in the context of elevation sounds a bit off. Alternatives to consider: fixed range height or fixed range size?

Reply:
@nirvn ok, i've renamed to fixed range *size*